aip/general/0233.md

```diff
@@ -114,7 +116,9 @@ message BatchCreateBooksResponse {
 
 ## Changelog
```

Comment:
nit: can you add a rationale section to explain why we need partial responses? or is it self-evident? 

Just thought it's a good time to get used to adding rationale for things: https://google.aip.dev/8#document-structure

Reply:
I like the idea of documenting the rationale but in this case I don't know if I can introduce an AIP-wide rationale section in this PR since it probably needs to explain the whole AIP. Let me know if you think otherwise and I can add it in a followup change.